Force change password (#3297)
aws-amplify/docs: docs/lib/auth/fragments/native_common/signin/common.md

Confirm signUp succeeded
```

## Force change password

> If a user is required to change the password on the first login, CONFIRM_SIGN_IN_WITH_NEW_PASSWORD `AuthSignInStep` will be returned when `signIn` is called. A new password given by the user is needed in that case. It can be done using `confirmSignIn` with the new password.

Reviewer: I looked up the codebase and I found that the signature of `confirmSignIn()` is:
```
public void confirmSignIn(
            @NonNull String confirmationCode,
            @NonNull Consumer<AuthSignInResult> onSuccess,
            @NonNull Consumer<AuthException> onException
    )
```
where the first parameter is `confirmationCode`. Why can we pass a password provided by user as a parameter to this method?

> If a user is required to change their password on first login, CONFIRM_SIGN_IN_WITH_NEW_PASSWORD AuthSignInStep will be returned when signIn is called. You need to provide a new password given by the user in that case. It can be done using confirmSignIn with the new password.

Reviewer: ```suggestion
If a user is required to change the password on the first login, CONFIRM_SIGN_IN_WITH_NEW_PASSWORD `AuthSignInStep` will be returned when `signIn` is called. A new password given by the user is needed in that case. It can be done using `confirmSignIn` with the new password.
```
